src/compositedataframe.jl

```diff
@@ -43,14 +43,18 @@ A constructor of an `AbstractCompositeDataFrame` that mimics the `DataFrame`
 constructor.  This returns a composite type (not immutable) that is an
 `AbstractCompositeDataFrame`.
 
-This uses `eval` to create a new type within the current module.
+This uses `eval` to create a new type within the module specified by the
+inmodule keyword.
 
 ### Arguments
 
 * `columns` : contains the contents of the columns
 * `cnames` : the names of the columns
 * `typename` : the optional name of the type created
 * `kwargs` : the key gives the column names, and the value is the column contents
+* `inmodule = DataFramesMeta` : a keyword argument to specify what module you
+want to define the type in. Consider passing `current_module()` or
```

Comment:
Align this line and the next to `` `inmodule` `` AFAIK. If you drop support for Julia < 0.6, no need to mention old Julia versions.